towerlib/entities/core.py

```diff
@@ -121,7 +121,7 @@ def validate_characters(value, alpha=True, numbers=True, extra_chars=None):
     valid_characters = '^[{alphas}{nums}{extra_characters}]+$'.format(alphas=alphas,
                                                                       nums=nums,
                                                                       extra_characters=extra_characters)
-    return True if re.search(valid_characters, value) else False
+    return re.search(valid_characters, value)
```

Comment:
re.search does not return a boolean, it returns a group if found None if not. This should be at least wrapped in a ```bool()```

Reply:
wrapped into boolean.